### DISCUSSION TITLE
Trash can icon visible when reordering assets

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#6665] [General]
Hi

When reordering assets, in the assets field the trash can icon is shown

<img width="1055" alt="Screenshot 2022-09-08 at 10 19 29" src="https://user-images.githubusercontent.com/72254741/189085743-0b6b932f-ba5e-4ab5-9714-5a7f30f8b65a.png">

```
Antlers: runtime
Version: 3.3.34 PRO
```

**Comment by duncanmcclean** [comment]
This is a bug - would be you be able to [open an issue](https://github.com/statamic/cms/issues/new/choose) instead of a discussion for this?

**Reply by jackmcdade** [reply]
Fixed in https://github.com/statamic/cms/pull/6671
